src/components/Main/Main.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './Main.css';
-
+import Animal from '../Animal/Animal';
+import { animals } from '../../data';
+import background from '../../background.png';
 export default function Main() {
-  return <main></main>;
+  return (
+    <main style={{ backgroundImage: `url(${background})` }}>
+      {animals.map((animal) => (
+        <Animal key={animal.name} animal={animal} />
+      ))}
```

Comment:
great use of the .map. this will be really handy as you build more robust React apps.